### PR TITLE
[[ Addons ]] Include addon license array in standalones

### PR DESCRIPTION
**Pull request by montegoulding**
This patch includes the addon license array in the standalone license
capsule and also adds community plus to the enum.

### CODE REVIEW

File: engine/src/deploy.cpp [hunk 1]
```diff
@@ -398,21 +398,67 @@ static bool MCDeployWriteDefineLicenseSection(const MCDeployParameters& p_params
 			t_edition = 1;
 			break;
 		
-		case kMCLicenseClassEvaluation:
```
Comment (runrevmark) [inline engine/src/deploy.cpp]: I know I wrote this code - however, I don't know why I didn't just use the values from the kMCLicenseClass enum... Can you see a reason why we can't just use (uint1)MClicenseparameter.license_class; and set it directly back in the capsule load?

Comment (montegoulding) [reply engine/src/deploy.cpp]: what type are enums? are they consistent for all our toolchains?

Comment (runrevmark) [reply engine/src/deploy.cpp]: Maybe not, but that's why I suggested casting... Something like...
```
IO_write_uint1(..., (uint1)MClicenseparameter.license_class);

...

uint1 t_class;
IO_read_uint1(..., &t_class);
MClicenseparameters.license_class = (MCLicenseClass)t_class
```

Comment (montegoulding) [reply engine/src/deploy.cpp]: ok, one less thing to update if we add more editions!

Comment (montegoulding) [reply engine/src/deploy.cpp]: OK, done... although by the look of the tests I've broken standalone building here.

Comment (montegoulding) [reply engine/src/deploy.cpp]: ah, found it I think

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 16f64b9

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 7ba41b6fbc011f70a78c572fab24b4151a905685

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 7ba41b6fbc011f70a78c572fab24b4151a905685

**Comment by livecode-vulcan** [comment]
😎 test success 7ba41b6fbc011f70a78c572fab24b4151a905685

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/681)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/697)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/671)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/614)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/617)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/675)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/695)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/680)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/697)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/679)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/690)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/712)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/676)

**Pull request merged**